### DISCUSSION TITLE
Storybook 6.5.16 built with Vite and deployed on Vercel stuck on loading state

### DISCUSSION THREAD

**Discussion by arantebw** [vercel/turborepo#4280] [Help]
Our project is a monorepo (using Turborepo) with UI library made with Storybook 6.5.16 and bundled by Vite. Everything was good running locally, so we tried to deploy it on Vercel so that the entire team can access the current design system, but it was stuck on loading state with the following error in the console:
![image](https://user-images.githubusercontent.com/8081121/226574803-53a195f4-e8d7-49e5-b246-07f7f0ff5a5c.png)
I think the build and deployment went well on Vercel, no errors, but I have no idea where to go from here with regards to this issue being stuck in the loading state. I kept on searching online for any related issue, but I cannot find any.

**Comment by anthonyshew** [comment]
A few questions so we can hopefully point you in the right direction:
- Does you also get this error when you build and start the project locally?
- Storybook is on version 7 today. Does bumping to latest help at all?
- Are you able to find an absolute reference in your code somewhere? Does changing it to relative resolve the issue?

If none of those help, a reproduction would be great so we can try things out ourselves. 🙏

**Comment by arantebw** [comment]
@anthonyshew I appreciate the response 🙏🏼 , but it has been way too old of an issue. I can't even recall how we managed to fix it, but thank you anyway.

I will now close this discussion.

**Reply by anthonyshew** [reply]
No problem and sorry about the delay.

We're doing better to keep up discussions now! Thanks for closing.
